haystack/pipelines/base.py

```diff
@@ -1905,36 +1911,36 @@ def load_from_config(
-            |               "type": "FARMReader",  # Haystack Class name for the component
-            |               "params": {"no_ans_boost": -10, "model_name_or_path": "deepset/roberta-base-squad2"},
-            |           },
-            |           {
-            |               "name": "MyESRetriever",
-            |               "type": "BM25Retriever",
-            |               "params": {
-            |                   "document_store": "MyDocumentStore",  # params can reference other components defined in the YAML
-            |                   "custom_query": None,
-            |               },
-            |           },
-            |           {"name": "MyDocumentStore", "type": "ElasticsearchDocumentStore", "params": {"index": "haystack_test"}},
-            |       ],
-            |       "pipelines": [
-            |           {  # multiple Pipelines can be defined using the components from above
-            |               "name": "my_query_pipeline",  # a simple extractive-qa Pipeline
-            |               "nodes": [
-            |                   {"name": "MyESRetriever", "inputs": ["Query"]},
-            |                   {"name": "MyReader", "inputs": ["MyESRetriever"]},
-            |               ],
-            |           }
-            |       ],
-            |   }
-            ```
+           ```python
+           {
+               "version": "ignore",
+               "components": [
+                   {  # define all the building-blocks for Pipeline
+                       "name": "MyReader",  # custom-name for the component; helpful for visualization & debugging
+                       "type": "FARMReader",  # Haystack Class name for the component
+                       "params": {"no_ans_boost": -10, "model_name_or_path": "deepset/roberta-base-squad2"},
+                   },
+                   {
+                       "name": "MyESRetriever",
+                       "type": "BM25Retriever",
+                       "params": {
+                           "document_store": "MyDocumentStore",  # params can reference other components defined in the YAML
+                           "custom_query": None,
+                       },
+                   },
+                   {"name": "MyDocumentStore", "type": "ElasticsearchDocumentStore", "params": {"index": "haystack_test"}},
+               ],
+               "pipelines": [
+                   {  # multiple Pipelines can be defined using the components from above
+                       "name": "my_query_pipeline",  # a simple extractive-qa Pipeline
+                       "nodes": [
+                           {"name": "MyESRetriever", "inputs": ["Query"]},
+                           {"name": "MyReader", "inputs": ["MyESRetriever"]},
+                       ],
+                   }
+               ],
+           }
+           ```
```

Comment:
Same
```suggestion
           ```python
           {
               "version": "ignore",
               "components": [
                   {  # define all the building-blocks for Pipeline
                       "name": "MyReader",  # custom-name for the component; helpful for visualization & debugging
                       "type": "FARMReader",  # Haystack Class name for the component
                       "params": {"no_ans_boost": -10, "model_name_or_path": "deepset/roberta-base-squad2"},
                   },
                   {
                       "name": "MyRetriever",
                       "type": "BM25Retriever",
                       "params": {
                           "document_store": "MyDocumentStore",  # params can reference other components defined in the YAML
                           "custom_query": None,
                       },
                   },
                   {"name": "MyDocumentStore", "type": "ElasticsearchDocumentStore", "params": {"index": "haystack_test"}},
               ],
               "pipelines": [
                   {  # multiple Pipelines can be defined using the components from above
                       "name": "my_query_pipeline",  # a simple extractive-qa Pipeline
                       "nodes": [
                           {"name": "MyRetriever", "inputs": ["Query"]},
                           {"name": "MyReader", "inputs": ["MyRetriever"]},
                       ],
                   }
               ],
           }
           ```
```